modules/ui/init.js

```diff
@@ -228,6 +228,7 @@ export function uiInit(context) {
 
         var mapDimensions = map.dimensions();
 
+
```

Comment:
fwiw, I've been adding 2 spaces before function blocks and anywhere else I want documentation to go.